server/pkg/provision/auth.go

```diff
@@ -43,3 +61,120 @@ func emailSettings(ctx context.Context, s store.Storer) error {
 		})
 	})
 }
+
+// addAuthSuperUsers assigns BYPASS roles to users from AUTH_PROVISION_SUPER_USER value
+// When in Production, Corteza should stop and report an error.
+func addAuthSuperUsers(ctx context.Context, log *zap.Logger, s store.Storer) (err error) {
+	var (
+		val, has = os.LookupEnv("AUTH_PROVISION_SUPER_USER")
+		envOpt   = options.Environment()
+	)
+
+	if !has {
+		return
+	}
+
+	if envOpt.IsProduction() {
+		return errors.New(0, "Cannot provision super users in Production environment!")
```

Comment:
Please be more instructive in regards to what the user tried to do and what they need to do to fix the issue -- in this case it's the dilemma with the different env variables.

Also, start error strings with a lower case character and don't punctuate it (especially not with `!`)